tests/Analytics/AnalyticsTest.php

```diff
@@ -31,6 +31,15 @@ public function testGoogleAnalytics()
         $this->assertTrue($this->ga->createPageView("appwrite.io", "/docs/installation"));
         $this->assertTrue($this->ga->createEvent("testEvent", "testEvent"));
 
+        $this->ga->disable();
+        $this->assertFalse($this->ga->createPageView("appwrite.io", "/docs/installation"));
+        $this->assertFalse($this->ga->createEvent("testEvent", "testEvent"));
+    }
+    public function testPlausible()
+    {
+        $this->assertTrue($this->ga->createPageView("appwrite.io", "/docs/installation"));
+        $this->assertTrue($this->ga->createEvent("testEvent", "testEvent"));
+
```

Comment:
Look's like you're still using the Google Analytics adapter in your tests. You need to write these tests after creating a new instance of the Plausible adapter.